source/features/download-folder-button.tsx

```diff
@@ -14,12 +32,11 @@ function add({parentElement: deleteDirectoryItem}: HTMLAnchorElement): void {
 	link.removeAttribute('aria-keyshortcuts');
 	link.removeAttribute('aria-labelledby');
 
-	deleteDirectoryItem!.before(item);
+	deleteDirectoryItem?.before(item);
 }
 
 function init(signal: AbortSignal): void {
-	// Selector points to "Delete directory" button
-	observe('a[aria-keyshortcuts="d"]', add, {signal});
+	observe('ul[role="menu"]', add, {signal});
```

Comment:
Let's just point to "copy permalink" or any other permanent link. We can't deal with two possibly-different DOMs.